### DISCUSSION TITLE
characters including Chinese cannot be read

### DISCUSSION THREAD

**Discussion by sparklog** [acheong08/ChatGPT#794] [Q&A]
hello there, 
When I use ask method, everything goes well. However, the return value which including Chinese characters of get_conversations or get_msg_history cannot be read. How could I get the correct characters?
thanks a lot

**Comment by acheong08** [comment]
Can you show the error and response? It could be an issue with Python's UTF-8 encoding

**Reply by rakuyoMo** [reply]
I also encountered this problem, and part of its return is shown below:
```
"è¯·ç»æåä¸ä¸ªæ¯è¾ç®åçswiftä»£ç "
```

<img width="344" alt="image" src="https://user-images.githubusercontent.com/16268770/219563661-e9e197f2-4baf-45d1-8c80-215eb4f0cf44.png">

I tried to execute the `encode` method first and then the `decode` method on it, but it still doesn't work, it doesn't seem to read properly.

**Reply by sparklog** [reply]
yesterday (Feb 17th) works well, however, today response issues again. During these time, I have not updated the code. Thus, perhaps something potential error in the server.
following is the response by print():
æ¯¬ç®ä» 

thanks a lot

**Reply by acheong08** [reply]
Chinese works perfectly fine for me. Could be an issue with your console or Python version

**Reply by rakuyoMo** [reply]
> Chinese works perfectly fine for me. Could be an issue with your console or Python version

I'm using version 3.11 of python, what version are you using? @sparklog @acheong08

**Reply by sparklog** [reply]
my python is 3.9.6 , and the system is macos 13.1
the return value isn't normal until today.

**Reply by rakuyoMo** [reply]
> my python is 3.9.6 , and the system is macos 13.1
> the return value isn't normal until today.

I still have problems on my side, what is the version of `revChatGPT` you are using?

**Reply by sparklog** [reply]
I have updated revChatGPT already, however,  the issue we talk about remains as well ;(

**Reply by acheong0** [reply]
Issue with Python, not revChatGPT. It should use utf-8 by default. Upgrade to Python 3.10

**Reply by rakuyoMo** [reply]
> Issue with Python, not revChatGPT. It should use utf-8 by default. Upgrade to Python 3.10

I'm using python version 3.11, but the problem persists. You are using 3.10 version of python without problems?

**Comment by rakuyoMo** [comment]
After upgrading to the latest [2.3.4](https://github.com/acheong08/ChatGPT/releases/tag/2.3.4) version, it was solved.
